### ISSUE TITLE
TestFramework: "network_connect_handshake_test" in BasicNetworkTestSuite fails in requiring handshake

### ISSUE THREAD

**Issue by sihammill** [YoYoGames/GameMaker-Bugs#8220]
### Description

Workflow artefact: https://github.com/YoYoGames/GM-TestFramework/actions/runs/11610157668#artifacts

Test code from Test Suite:
```
ev_web_networking: function() {
			
	var _asyncLoad = async_load;
	
	if (_asyncLoad[? "id"] != socket) exit;
	
	if (_asyncLoad[? "type"] == network_type_non_blocking_connect) {
		assert_true(_asyncLoad[? "succeeded"], "#1 network_connect(), fail to require handshake");
		
		network_destroy(socket);
		
		test_end();
	}
}
```

_You can jump directly to the current location of this code via this [link](https://github.com/YoYoGames/GM-TestFramework/blob/85e03a71e87b23cffb3b93b42b18081307cf2f09/projects/xUnit/scripts/BasicNetworkTestSuite/BasicNetworkTestSuite.gml#L32)_

Logged result:
```
Testsuite Name: BasicNetworkTestSuite
Test Name: network_connect_handshake_test
description: #1 network_connect(), fail to require handshake
title: Asserted value to be true
stack: gml_Script_anon@934@___struct___126@BasicNetworkTestSuite@BasicNetworkTestSuite:40
actual: 0.0
expected: null
```

### Steps To Reproduce

No repro steps available, this is a test run from the test suite stated above.

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.1100 (Betas)

### Which platform(s) are you seeing the problem on?

Windows

### Sample Package Attached?

- [ ] I have attached my bug package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by DiasFranciscoA** [comment]
This was a problem with the Test in the TestFramework and should now be fixed with commit SHA: 1cf114224a174aa391011ee8609d50e898340ed3

**Comment by sihammill** [comment]
Fix verified in IDE v2024.1100.0.681  Runtime v2024.1100.0.703, the test now passes as expected in the Test Framework
